src/components/FormInput/FormInput.js

```diff
@@ -0,0 +1,37 @@
+import { useState } from 'react';
+import styled from 'styled-components';
+
+export default function FormInput(props) {
+  const [focused, setFocused] = useState(false);
+  const { label, errorMessage, onChange, id, pattern, value, ...inputProps } =
+    props;
+
+  return (
+    <section aria-labelledby="inputlabel">
+      <label id="inputlabel">{label}</label>
+      <StyledInput
+        {...inputProps}
+        onChange={onChange}
+        onBlur={handleFocus}
+        focused={focused.toString()}
+        value={value}
+      />
```

Comment:
The label and input element are not connected. I would recommend to connect them as you usually do with label and input elements :)
```suggestion
      <label htmlFor={name}>{label}</label>
      <StyledInput
         id={name}
        {...inputProps}
        onChange={onChange}
        onBlur={handleFocus}
        focused={focused.toString()}
        value={value}
      />
```
For this to work you would need to destructure `name` as well from your props.